conslogging/bundle.go

```diff
@@ -198,9 +198,9 @@ func (tl *targetLogger) toManifestTarget() (TargetManifest, error) {
 		return TargetManifest{}, errors.New("0 length target")
 	}
 
-	if tl.prefix == "ongoing" {
-		// The ongoing messages end up in here too. Since they are not updates from a vertex, we will never mark them as complete.
-		// Additionally, its not useful to have in the output. Ignore it here.
+	if tl.prefix == "ongoing" || tl.prefix == "buildkitd" {
+		// The ongoing & buildkitd init messages end up in here too. Since they are not updates from a vertex, we will
+		// never mark them as complete. Additionally, its not useful to have in the output. Ignore it here.
```

Comment:
I believe this stuff will get better when we switch to using the sts ID's as target IDs. Anything that doesn't have an sts ID isn't a real target, so we can ignore it automatically.

Reply:
I would love if we had a way to include out of band things as "targets", though